LoadTest/Models/Company/Requests/Company/Filters/GetCompanyFilter.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalOffice.LoadTesting.Models.Company.Requests.Company.Filters
+{
+    public record GetCompanyFilter
+    {
+        public bool? IncludeDepartments { get; set; } = true;
```

Comment:
why bool is nullable if you set the default value?

Reply:
default value is for me, but we use this model in company service, and all this models are from ours services